### PR TITLE
build: prod build to bundle every thing together

### PR DESCRIPTION
**Pull request by yqrashawn**
PORTAL-1632
PORTAL-1461
PORTAL-974

---
This change is [Reviewable](https://reviewable.io/reviews/conflux-chain/helios/334)

### CODE REVIEW

**Comment by angelia-yuqi-personal** [comment]
Task linked: [PORTAL-1461 修复 Bug: can't unzip by chrome on windows os](https://app.clickup.com/t/3719212/PORTAL-1461)

**Comment by angelia-yuqi-personal** [comment]
Task linked: [PORTAL-1632 js 和 css 分开打包，js 不引入 css，尤其是组件库](https://app.clickup.com/t/3719212/PORTAL-1632)

**Comment by angelia-yuqi-personal** [comment]
Task linked: [PORTAL-974 \background\sp_\pkg\common目录下的_polyfill-node:global-acbc543a.js文件名中有冒号，导致解压缩后不能正常运行](https://app.clickup.com/t/3719212/PORTAL-974)

File: scripts/snowpack.base.config.js [hunk 1]
```diff
@@ -78,11 +78,13 @@ module.exports = {
   optimize: isProd()
     ? {
         souecemap: 'inline',
-        // bundle: true, // TODO: turn this on after https://github.com/snowpackjs/snowpack/issues/3403 is resolved
+        // TODO: turn this on after https://github.com/snowpackjs/snowpack/issues/3403 is resolved
```
Comment (angelia-yuqi-personal) [inline scripts/snowpack.base.config.js:81]: seems it has been turned on?

File: packages/popup/src/App.js [hunk 2]
```diff
@@ -5,7 +5,6 @@ import {HashRouter as Router, Redirect, Route, Switch} from 'react-router-dom'
 import {useIsLocked, useIsZeroGroup, usePendingAuthReq} from './hooks/useApi'
 import {ProtectedRoute} from './components'
 import {ROUTES} from './constants'
-import './index.css'
```
Comment (angelia-yuqi-personal) [inline packages/popup/src/App.js:8]: When do the components css separation

**Review by angelia-yuqi-personal — CHANGES_REQUESTED** [review]

File: packages/popup/src/App.js (hunk 2 above)
Comment (yqrashawn) [reply packages/popup/src/App.js:8]: there's a dup one in app.js

File: scripts/snowpack.base.config.js (hunk 1 above)
Comment (yqrashawn) [reply scripts/snowpack.base.config.js:81]: it's resolved in @yqrashawn/snowpack

Comment (angelia-yuqi-personal) [reply scripts/snowpack.base.config.js:81]: But still wait the snowpack issue resolving. it's only a temporary workaround

File: packages/popup/src/App.js (hunk 2 above)
Comment (yqrashawn) [reply packages/popup/src/App.js:8]: https://github.com/Conflux-Chain/helios/blob/2aa0159e4402c4b66cc4421b546bde98f37ac036/packages/popup/src/index.js#L3

**Review by angelia-yuqi-personal — APPROVED** [review]

**Pull request merged**